src/main/java/fi/oph/akt/repository/AuthorisationTermRepository.java

```diff
@@ -19,8 +19,10 @@
   @Query(
     "SELECT at.id" +
     " FROM AuthorisationTerm at" +
+    " JOIN at.authorisation a" +
+    " JOIN a.translator t" +
     " LEFT JOIN at.reminders atr" +
-    " WHERE at.endDate BETWEEN ?1 AND ?2" +
+    " WHERE at.endDate BETWEEN ?1 AND ?2 AND t.email IS NOT NULL" +
```

Comment:
Sinänsä tietysti ihan hyvä ettei yritetä lähettää mailia umpeutuvista auktorisoinneista kääntäjille joiden maili ei ole tiedossa. Nostaa toki esille kysymyksen siitä mikä prosessi on tästä eteenpäin: oletetaanko kääntäjän muistavan ottaa yhteyttä proaktiivisesti vai oletetaanko virkailijoiden pitävän aktiivisesti silmällä umpeutuvien listaa?

Reply:
Muistaakseni juteltiin tästä aiemmin joko dailyssä tai @terova kanssa kahdestaan, että virkailijat huolehtisivat siitä, että kääntäjien sähköpostiosoitteet ovat tiedossa ja saavat näin tiedot umpeutumisista. Tästä pitäisi tosin heiltä kysyä että miten haluavat.